packages/lesswrong/components/dropdowns/comments/CommentSubscriptionsDropdownItem.tsx

```diff
@@ -35,6 +36,14 @@ const getNotifyMeItems = ({comment, post, currentUser, enableSubscribeToCommentU
     title: `New replies to this comment`,
     subscriptionType: subscriptionTypes.newReplies,
   },
+  {
+    document: comment.user,
+    enabled: enableSubscribeToCommentUser && isAdmin(currentUser),
```

Comment:
this should probably be feature-gated, not just admin-gated (so the EA forum admins don't see it)